CHANGES.md

```diff
@@ -4,6 +4,18 @@ This file is only intended to serve the developers of this repository and of the
 
 This provides an overview of the releases that have been made using this release pipeline.
 
+
+## November 2023: First 1.40 release
+
+The [standard MediaWiki upgrade process](https://www.mediawiki.org/wiki/Manual:Upgrading) can be followed for this release.
+
+- [MediaWiki release notes](https://www.mediawiki.org/wiki/Release_notes/1.40)
+- [Wikibase release notes](https://github.com/wikimedia/Wikibase/blob/REL1_40/RELEASE-NOTES-1.40)
+
+| Suite Version | Version Info | Date available | Run number | Release task  |
+|---------------|----------------------|----------------|------------| --------------|
+| wmde.14 ([spec](https://github.com/wmde/wikibase-release-pipeline/blob/wmde.14/variables.env)) | MediaWiki (1.40.1, PHP 8.1.24), WDQS (0.3.135)    | 10 November 2023 | [TODO](https://github.com/wmde/wikibase-release-pipeline/actions/runs/TODO) | [T340939](https://phabricator.wikimedia.org/T340939) |
```

Comment:
In the last release we added the apache version.  
I just checked and it's again: debian/apache2 2.4.57-2

(see https://hub.docker.com/layers/library/mediawiki/latest/images/sha256-8461b941852c8a22475753f49971f8d4b0b452f4ef3bf979912e291f9b9767fb?context=explore for more info)

Reply:
is there a reason why you left it out?

Reply:
No, thanks for checking.
Added.